portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,75 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.List;
+import com.google.gson.Gson;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+
+class Comment {
+  String message;
+  long timestamp;
+
+  Comment(String messageIn, long timestampIn){
+      message = messageIn;
+      timestamp = timestampIn;
+  }
```

Comment:
```suggestion
  Comment(String message, long timestamp){
    this.message = message;
    this.timestamp = timestamp;
  }
```
+2 indents and you can remove postfix In by that.